pom.xml

```diff
@@ -92,11 +92,12 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <inceptionYear>2017</inceptionYear>
         <org.slf4j.version>1.7.30</org.slf4j.version>
-        <jetty.version>9.4.19.v20190610</jetty.version>
+        <jetty.version>9.4.38.v20210224</jetty.version>
         <jax.rs.api.version>2.1</jax.rs.api.version>
         <jersey.server.version>2.29.1</jersey.server.version>
         <jersey.client.version>2.28</jersey.client.version>
         <jackson.version>2.10.3</jackson.version>
+        <jackson-databind.version>2.9.10.8</jackson-databind.version>
```

Comment:
Is there a reason for not updating both `jackson-core` and `jackson-databind` to a more recent version?  Both libraries have version `2.12.2` available as of 2021-03-03.  At minimum, it seems like it would be best to keep the minor version the same between `jackson-core` and `jackson-databind` as opposed to having `2.10` for `jackson-core` and `2.9` for `jackson-databind`.

Reply:
Could we possibly get rid of `jackson-databind.version` now? I think we didn't originally have that, but at one point there was a special fourth digit release of `jackson-databind` that didn't exist for `jackson-core`. Seems like now we could possibly get them back in sync.

Reply:
@exceptionfactory @bbende Agreed both `jackson-core` and `jackson-databind` can now be on the same minor version. Reverted back to use one `jackson.version` and bumped the version. Thanks.

Reply:
@bbende I'll see what I can do to remove the 'jackson-databind' version property for NiFi.